client/discovery/external/testdata/good.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require "json"
+require "pp"
+
+if ARGV.length != 1
```

Comment:
this is just to test the arg parsing works, and probably not even a good test, wouldnt be needed normally